test/utils/nrosched/nrosched.go

```diff
@@ -59,6 +57,9 @@ func checkPODEvents(k8sCli *kubernetes.Clientset, podNamespace, podName, schedul
 		klog.ErrorS(err, "cannot get events for pod %s/%s", podNamespace, podName)
 		return false, err
 	}
+	if len(events.Items) == 0 {
+		klog.Warningf("No events received for %s/%s!", podNamespace, podName)
```

Comment:
and we should likely `return false` here. Or we may go as far as return `false, fmt.Errorf("no event received for %s/%s", podNamespace, podName)`

Reply:
Ack